posts/Guillaume/2017-06-29-hibernate-validator-600-cr1-out.adoc

```diff
@@ -0,0 +1,43 @@
+= "Hibernate Validator 6.0.0.CR1 is out with Bean Validation 2.0.0.CR1 support"
+Guillaume Smet
+:awestruct-tags: [ "Hibernate Validator", "Releases" ]
+:awestruct-layout: blog-post
+---
+The http://beanvalidation.org/news/2017/06/26/bean-validation-2-0-proposed-final-draft-released/[Proposed Final Draft (CR1) of Bean Validation 2.0] has been released earlier this week and it is time to release a version of the Reference Implementation compatible with it: here comes Hibernate Validator 6.0.0.CR1.
+
+Note that Hibernate Validator 6 requires JDK 8 or above.
+
+== What's new since Beta2
+
+Not much to be honest. As we polished the specification and the TCK, we fixed several bugs - mostly corner cases - and improved overall consistency.
```

Comment:
That firs sentence sounds a bit negative. Maybe something like "As the final release of Bean Validation is getting closer, we mainly focused on keeping up with the latest spec clarifications, fixing bugs ... ".

Also a link to all the issues -- https://hibernate.atlassian.net/projects/HV/versions/29202/tab/release-report-all-issues -- should be added.